avm/utilities/pipelines/e2eValidation/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -185,6 +185,43 @@ function Invoke-ResourceRemoval {
             }
             break
         }
+        'Microsoft.Subscription/aliases' {
+            #Update-Module -Name Az.Accounts -Force
+            #Update-Module -Name Az.Subscription -Force
+            #Import-Module Az.Subscription -Force
+
+            $subscriptionName = $ResourceId.Split('/')[4]
+            $subscription = Get-AzSubscription | Where-Object {$_.Name -eq $subscriptionName}
+            $subscriptionId =  $subscription.Id
+            $subscriptionState = $subscription.State
+
+            $null = Select-AzSubscription -SubscriptionId $subscriptionId -WarningAction SilentlyContinue
+
+            if ($PSCmdlet.ShouldProcess("Subscription [$subscriptionName]", 'Remove')) {
```

Comment:
Mind you, the `ShouldProcess` should only wrap individual state-changing commands so that you can tell what the script 'would' actually do as opposed to the entire block 😉